.travis.yml

```diff
@@ -80,6 +80,8 @@ deploy:
           - dist/*.rpm
           - dist/*.tar.gz
       skip_cleanup: true
+      provider: script
```

Comment:
Does this line need a `-` for a yaml array?

Reply:
@bryphe I don't think see, but I only have a cursory understanding of travis' yaml configs. I was going off of [this tectonic config file](https://github.com/tectonic-typesetting/tectonic/blob/a5fd0f700bb56c87ed9ea970d3846e0dfaa9020f/.travis.yml#L102) where they're also deploying to AUR.

Reply:
Whoops, nvm. I see what you're saying. Changing now.